src/re/main.c

```diff
@@ -124,6 +124,7 @@ lang_name(const char *name, enum fsm_print_lang *fsm_lang, enum ast_print_lang *
 		{ "rust",       FSM_PRINT_RUST       },
 		{ "sh",         FSM_PRINT_SH         },
 		{ "vmc",        FSM_PRINT_VMC        },
+		{ "cdata",      FSM_PRINT_CDATA      },
```

Comment:
and rx, and retest too please!

Reply:
rx: Added in 3ea08983.

retest: Added in 7a935b4e, and that found a couple bugs because it exercised cdata output more widely.